.github/workflows/e2e-component-tests-embedding-sdk-cross-version.yml

```diff
@@ -127,6 +127,116 @@ jobs:
               sdk_version,
             };
 
+  e2e-tests:
```

Comment:
This is because Dan removed the whole workflow last week as it was failing in the 52 branch. This PR restores the workflow as-is.